src/main/app.js

```diff
@@ -355,9 +335,13 @@ fluid.defaults("gpii.app", {
             funcName: "gpii.app.exit",
             args: "{that}"
         },
-        "handleUncaughtException": {
+        handleUncaughtException: {
```

Comment:
This could go into the dedicated "errors" component along with the descriptionMap